src/amber/cli/generators/mailer.cr

```diff
@@ -2,6 +2,7 @@ require "./field.cr"
 
 module Amber::CLI
   class Mailer < Generator
+    command :mailer
```

Comment:
I think this is what @elorest means `class_getter registered_commands = :mailer`

Reply:
@eliasjpr take a look at the conversation with @Sija .  We could simplify this using the class name as the generator name instead of using this macro.  I kinda like explicitly specifying the command.  I was also thinking of adding some type of description for help.